core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -242,20 +242,14 @@ public void repairTable(
       throw new IllegalArgumentException(
           "The table " + getFullTableName(namespace, table) + "  does not exist");
     }
-    // The table metadata are not managed by ScalarDB, so we don't need to do anything
-    // here
+    // The table metadata are not managed by ScalarDB, so we don't need to do anything here
   }
 
   @Override
   public void addNewColumnToTable(
       String namespace, String table, String columnName, DataType columnType)
       throws ExecutionException {
     try {
-      if (getTableMetadata(namespace, table).getColumnNames().contains(columnName)) {
```

Comment:
Since we have checks in `CheckedDistributedStorageAdmin`, removed these checks from the admin implementations.